slideshow.md

```diff
@@ -529,24 +529,230 @@ name: presentation
 layout: true
 class: middle
 
-Fandango - the Swiss army knife
-===============================
+Fandango - a Swiss army knife for tango
```

Comment:
I see the title ends up on every slide - is that what you want?
(I'm not sure how to change it!)